webapp/src/containers/components/tables/existingConfigTable.tsx

```diff
@@ -42,162 +42,154 @@ const ExistingConfigTable = ({value, onChange}: Props) => {
-                                                    <OverlayTrigger
-                                                        placement='top'
-                                                        overlay={<Tooltip>{'Edit config'}</Tooltip>}
-                                                    >
-                                                        <Button onClick={() => handleEdit(i)}>
-                                                            <svg
-                                                                className='svg'
-                                                                xmlns='http://www.w3.org/2000/svg'
-                                                                width='20'
-                                                                height='20'
-                                                                viewBox='0 0 24 24'
-                                                                fill='none'
-                                                                stroke='#333'
-                                                                strokeWidth='1.65'
-                                                                strokeLinecap='round'
-                                                                strokeLinejoin='round'
-                                                            ><path d='M20 14.66V20a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V6a2 2 0 0 1 2-2h5.34'/><polygon points='18 2 22 6 12 16 8 16 8 12 18 2'/></svg>
-                                                        </Button>
-                                                    </OverlayTrigger>
-                                                    <OverlayTrigger
-                                                        placement='top'
-                                                        overlay={<Tooltip>{'Delete Config'}</Tooltip>}
-                                                    >
-                                                        <Button
-                                                            variant='light'
-                                                            onClick={() => handleDelete(i)}
+                                                            <Button
+                                                                onClick={() => handleView(i)}
+                                                            >
+                                                                <svg
+                                                                    xmlns='http://www.w3.org/2000/svg'
+                                                                    width='20'
+                                                                    height='20'
+                                                                    viewBox='0 0 24 24'
+                                                                    fill='none'
+                                                                    stroke='#333'
```

Comment:
```suggestion
                                                                    stroke='#333333'
```
use hex

Reply:
what is the difference between #000 & #000000 ?
Raghav suggests using #000